Readability.js

```diff
@@ -1826,13 +1842,15 @@ Readability.prototype = {
     }
 
     var textContent = articleContent.textContent;
-    return { uri: this._uri,
-             title: articleTitle,
-             byline: metadata.byline || this._articleByline,
-             dir: this._articleDir,
-             content: articleContent.innerHTML,
-             textContent: textContent,
-             length: textContent.length,
-             excerpt: metadata.excerpt };
+    return {
+      uri: this._uri,
+      title: articleTitle,
+      byline: metadata.byline || this._articleByline,
+      dir: this._articleDir,
+      content: articleContent.innerHTML,
+      textContent: textContent,
+      length: textContent.length,
+      excerpt: metadata.excerpt
+    };
```

Comment:
I updated here because of the [eslint failures][eslint-failures]. The root cause is mentioned [here].
```
/home/travis/build/mozilla/readability/Readability.js
  1846:14  error  Expected indentation of 6 spaces but found 13  indent
  1847:14  error  Expected indentation of 6 spaces but found 13  indent
  1848:14  error  Expected indentation of 6 spaces but found 13  indent
  1849:14  error  Expected indentation of 6 spaces but found 13  indent
  1850:14  error  Expected indentation of 6 spaces but found 13  indent
  1851:14  error  Expected indentation of 6 spaces but found 13  indent
  1852:14  error  Expected indentation of 6 spaces but found 13  indent
```

[eslint-failures]: https://travis-ci.org/mozilla/readability/builds/172231876#L397-L404
[here]: https://github.com/mozilla/readability/pull/310#issuecomment-257528601